### PR TITLE
Localization: Fix cancel button in document repair dialog

### PR DESCRIPTION
**Pull request by pedropintosilva**
Allow Cancel button to be translated

Signed-off-by: Pedro Pinto Silva <pedro.silva@collabora.com>
Change-Id: I4e74bf23bea0edba5dad13f53967d380567dbdc7

### CODE REVIEW

File: browser/src/control/Control.DocumentRepair.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ L.Control.DocumentRepair = L.Control.extend({
 										{
 											id: 'cancel',
 											type: 'pushbutton',
-											text: '~Cancel',
+											text: _('~Cancel'),
```
Comment (timar) [inline browser/src/control/Control.DocumentRepair.js:61]: Please remove the ~, it is removed programmatically anyway, but then we'd get translations from Cancel for free.

Comment (timar) [reply browser/src/control/Control.DocumentRepair.js:61]: And a few lines below, make OK tanslatable, e.g. in Spanish it's Acceptar.
_('OK')
And actually the dialog title is untranslatable. too: 'Repair Document' :)

**Review by timar — CHANGES_REQUESTED** [review]

**Comment by timar** [comment]
... and `text: 'You haven\'t done anything to rollback yet...',`
etc., etc.
I don't know how, but this whole dialog was left out of localization.

**Review by timar — APPROVED** [review]
I made the changes that I requested.

**Comment by pedropintosilva** [comment]
Thanks @timar I force pushed again because there was a typo on 'ok' that was breaking tests:

```diff
										type: 'pushbutton',
-											text: _('OK)',
+											text: _('OK'),
 											enabled: true,
```

**Pull request merged**